src/decaylanguage/utils/errors.py

```diff
@@ -10,7 +10,5 @@
 
 
 class LineFailure(RuntimeError):
-    def __init__(self, line, message, *args, **kwargs):
-        super(LineFailure, self).__init__(
-            "{}: {}".format(line, message), *args, **kwargs
-        )
+    def __init__(self, line, message):
+        super(LineFailure, self).__init__("{}: {}".format(line, message))
```

Comment:
RuntimeError doesn't actually have any other args, so MyPy complains. 50/50 on having it check this, sometimes it's wrong, like with Mixins. But here it's fine.